CSharpSourceCode/Utilities/TOWCommon.cs

```diff
@@ -36,7 +36,13 @@ public static string GetRandomScene()
             if (Directory.Exists(path))
             {
                 var dirnames = Directory.GetDirectories(path);
-                filterednames = dirnames.Where(x => x.Contains("towmm_")).ToList();
+                filterednames = dirnames.Where(x =>
```

Comment:
in principle, I would like , if we can create smaller  commits, and have a better overview of merged commits..... or try to make seperate branches for bug hunting and feature implementation. I dont know if that is possible in our public repository but I think this can be preferable. the bug fix of the scene loading came here out of nothing.

Reply:
i agree with @Z3rca